### PR TITLE
[Form] Minor

### PR DESCRIPTION
**Pull request by ThomasLandauer**
Removing "starting"

### CODE REVIEW

File: form/data_transformers.rst [hunk 1]
```diff
@@ -8,8 +8,8 @@ Data transformers are used to translate the data for a field into a format that
 be displayed in a form (and back on submit). They're already used internally for
 many field types. For example, the :doc:`DateType </reference/forms/types/date>` field
 can be rendered as a ``yyyy-MM-dd``-formatted input text box. Internally, a data transformer
-converts the starting ``DateTime`` value of the field into the ``yyyy-MM-dd`` string
-to render the form, and then back into a ``DateTime`` object on submit.
+converts the ``DateTime`` value of the field to the ``yyyy-MM-dd`` string
```
Comment (xabbuh) [inline form/data_transformers.rst:11]: Into is correct, isn’t it?

**Review by xabbuh — CHANGES_REQUESTED** [review]

File: form/data_transformers.rst (hunk 1 above)
Comment (ThomasLandauer) [reply form/data_transformers.rst:11]: Looks like "to" and "into" are both common in general, but Symfony docs seem to prefer "to" usually.
=> @javiereguiluz please decide.

Comment (javiereguiluz) [reply form/data_transformers.rst:11]: I asked on Symfony Slack about this to native English speakers. They agreed on using "convert to", but suggested other tweak: https://github.com/symfony/symfony-docs/commit/dfc2982e1712f8b653bf426628a08c24819a845e

**Comment by javiereguiluz** [comment]
Thanks!

File: form/data_transformers.rst (hunk 1 above)
Comment (bizmate) [reply form/data_transformers.rst:11]: "Use “into” to describe where something is: going inside something else" -> Please use `to`

**Pull request merged**